### PR TITLE
Lots of BS4 Fixes (#3633)

### LINKED ISSUE #3633: Update BS4 RTL to 4.5.0
The editor of the BS4 RTL that we use just updated to 4.5.0. Will be creating a pull to update this soon.

### PR DESCRIPTION
**Pull request by tywrenn**
Fixes #3633

#### Short description of what this resolves:
- This updates BS4 RTL to 4.5.0 and does the below updates to Bootstrap & other areas

#### Changelog

- Update BS4 RTL to 4.5.0
- Huge Superhero Theme Fixes - Fixes colors not showing correctly in some areas, there's still a lot to fix here
- Fix add_edit_event cancel button glitch where it doesn't work
- JS Linted some areas
- Fixed find_code_dynamic (for now)
- Removes aquabg.gif as it is very old code that needs to be removed
- Adds body-color variable to use when needed
- Adds other tags to core header that were needed for BS4 that I forgot to add
- Removed unnecessary code & other fixes

### CODE REVIEW

**Comment by tywrenn** [comment]
@bradymiller Ready for testing

File: interface/patient_tracker/patient_tracker.php [hunk 1]
```diff
@@ -977,7 +967,7 @@ function (data) {
                 }).done(
                     function (data) {
                         $("#flb_selectors").slideToggle();
-                        $("#flb_caret").css('color', 'var(--danger)');
+                        $("#flb_caret").addClass('text-danger');
```
Comment (tywrenn) [inline interface/patient_tracker/patient_tracker.php:970]: Now that I look at this it needs to change cuz it wouldn't really work

**Comment by bradymiller** [comment]
hi @tywrenn , Looks good and appears to be testing well. Do you want this to go into codebase now or need to wait (regarding the comment about the addClass stuff)?

**Comment by tywrenn** [comment]
@bradymiller Still need to fix the addClass -- the reason it won't work is because it could add 2 text color classes which would conflict with one another

**Comment by tywrenn** [comment]
Fixed the issue now ready for testing/merging

**Comment by bradymiller** [comment]
@tywrenn , Code looks good and bringing in (fixed a minor conflict). Thanks for the improvements! -brady

**Pull request merged**